src/crafting.cpp

```diff
@@ -1216,6 +1216,17 @@ int recipe::batch_time(int batch) const
         return time * batch;
     }
 
+    // NPCs around you should assist in batch production if they have the skills
+    int assistants = 0;
+    if (!g->active_npc.empty()) {
```

Comment:
You really don't need this kind of test, the loop will work properly even if `active_npc` is empty.